yamllint/rules/document_start.py

```diff
@@ -94,6 +98,26 @@ def check(conf, token, prev, next, nextnext, context):
             yield LintProblem(token.start_mark.line + 1, 1,
                               'missing document start "---"')
 
+        if (conf['max-empty-lines-after'] >= 0 and
+                isinstance(prev, yaml.DocumentStartToken)):
+
```

Comment:
In the rest of the code we don't insert blank lines after `if` blocks... Maybe remove it? (same line 113)

Reply:
Done.